src/main/java/net/coderbot/iris/pipeline/DeferredWorldRenderingPipeline.java

```diff
@@ -927,6 +928,8 @@ public void finalizeLevelRendering() {
 		phase = WorldRenderingPhase.NONE;
 		programStackLog.clear();
 
+		centerDepthSampler.sampleCenterDepth();
```

Comment:
We aren't sampling center depth in the right place on any Iris version it seems. I checked ShadersMod and it samples center depth before it renders the hand (which depends on whether or not you're holding something translucent). Though, that has the notable shortcoming that the center depth is never sampled if the hand isn't rendered - we need more robust logic than that.

Reply:
Not sure how to handle this - we should approach this in a separate PR alongside the next one.